src/main/java/cms/gov/madie/measure/utils/JsonUtil.java

```diff
@@ -391,30 +390,13 @@ private static void handlePopulationValues(
               .build();
       populationValues.add(populationValue);
     }
-    if (population.get("DENOM") != null) {
-      TestCasePopulationValue populationValue =
-          TestCasePopulationValue.builder()
-              .name(PopulationType.DENOMINATOR)
-              .expected(population.get("DENOM").asInt())
-              .build();
-      populationValues.add(populationValue);
-    }
-    if (population.get("DENEX") != null) {
-      TestCasePopulationValue populationValue =
-          TestCasePopulationValue.builder()
-              .name(PopulationType.DENOMINATOR_EXCLUSION)
-              .expected(population.get("DENEX").asInt())
-              .build();
-      populationValues.add(populationValue);
-    }
-    if (population.get("NUMER") != null) {
-      TestCasePopulationValue populationValue =
-          TestCasePopulationValue.builder()
-              .name(PopulationType.NUMERATOR)
-              .expected(population.get("NUMER").asInt())
-              .build();
-      populationValues.add(populationValue);
-    }
+
+    fetchObservationValuesForCV(population, populationValues, measure);
+
+    fetchDenominatorValues(population, populationValues, measure);
+
+    fetchNumeratorValues(population, populationValues, measure);
```

Comment:
same as above - can we please rename this function

Reply:
@nmorasb updateTestCasePopulationNumeratorValues?